### PR TITLE
Base for tool integration

### PR DESCRIPTION
**Pull request by Chemin1**
These are some fixes necessary for integration of the merge library into the tools

### CODE REVIEW

File: src/include/kdbmerge.h [hunk 1]
```diff
@@ -10,11 +10,12 @@
 #define KDBMERGE_H_
 
 #include "kdb.h"
-#include "kdberrors.h"
 #include "kdbprivate.h"
 
 #ifdef __cplusplus
 extern "C" {
+using Key = ckdb::Key;
```
Comment (markus2330) [inline src/include/kdbmerge.h]: using in header files is bad practice, please avoid.

**Comment by Chemin1** [comment]
jenkins build libelektra please

**Comment by Chemin1** [comment]
jenkins build libelektra please

File: doc/news/_preparation_next_release.md [hunk 2]
```diff
@@ -344,6 +344,7 @@ you up to date with the multi-language support provided by Elektra.
 - Release notes now use git's union merge driver. _(Dominic Jäger)_
 - Please remove me. I'm only here for the build server. _(Dominic Jäger)_
 - I'm only here for the build server. This PR contains only fix ups. _(Dominic Jäger)_
+- Please remove me. I'm only here for the buld server. _(Dominic Jäger)_
```
Comment (sanssecours) [inline doc/news/_preparation_next_release.md]:
```suggestion
- Please remove me. I'm only here for the build server. _(Dominic Jäger)_
```

😄

**Comment by raphi011** [comment]
Is this PR being built? If I click on the jenkins `Details` link it pipeline doesn't show up and it is not listed as active.

**Comment by markus2330** [comment]
I am afraid the build server is currently very much overloaded. I hope we get the hetzner build agent soon back or migrate to the new server (@Mistreated?)

**Comment by raphi011** [comment]
> I am afraid the build server is currently very much overloaded. I hope we get the hetzner build agent soon back or migrate to the new server (@Mistreated?)

imo this is *really* urgent, the flakiness and slowness of the tests make it hard if not impossible to do any changes if you have to wait this long to verify them.

**Comment by Chemin1** [comment]
jenkins build libelektra please

**Comment by Chemin1** [comment]
jenkins build libelektra please

**Comment by markus2330** [comment]
@raphi011 you should be able to finish the pkg-config files now.

Link to builds https://build.libelektra.org/jenkins/job/libelektra/view/change-requests/job/PR-3234/

**Comment by raphi011** [comment]
> @raphi011 you should be able to finish the pkg-config files now.
> 
> Link to builds https://build.libelektra.org/jenkins/job/libelektra/view/change-requests/job/PR-3234/

lets see if the build succeeds https://github.com/ElektraInitiative/libelektra/pull/3241

**Pull request merged**